rcl/test/CMakeLists.txt

```diff
@@ -63,6 +63,17 @@ function(test_target_function)
     AMENT_DEPENDENCIES ${rmw_implementation}
   )
 
+  rcl_add_custom_gtest(test_get_node_names${target_suffix}
+    SRCS rcl/test_get_node_names.cpp
+    ENV
+      ${extra_test_env}
+      EMPTY_TEST=
+      NORMAL_TEST=foo
```

Comment:
Another unrelated question: why are these tests run for each rmw implementation?